public/stylesheets/style.css

```diff
@@ -0,0 +1,65 @@
+body {
+  font: 14px "Lucida Grande", Helvetica, Arial, sans-serif;
+  margin: 0;
+  min-width: 960px;
+  border-top: 10px solid #b71f39;
+
+}
+
+.grapes {
+  width: 100%;
+  height: 100%;
+}
+
+.searchWrapper {
+  margin-left: 5%;
+  display: inline;
+  justify-content: center;
+}
+
+.searchBox{
```

Comment:
Maybe consider moving to the right of the page? Also, will you be using media breakpoints to ensure mobile readiness?